test/testinfra.py

```diff
@@ -343,11 +367,18 @@ def scan(self, update, context, except_context=False):
         results = []
         master_contexts = []
         pull_contexts = []
+        verify_contexts = []
+        image_contexts = []
         for (context, what) in contexts.items():
```

Comment:
I think you understand it correctly.

The extra work/traffic created by also fetching closed pull requests is not enourmous.  We don't get statuses or labels for all of them.

But yes, we can look for a better way.  Maybe a good old cron job or systemd timer is better?